src/devices/tpms_citroen.c

```diff
@@ -12,6 +12,8 @@
 Citroen FSK 10 byte Manchester encoded checksummed TPMS data
 also Peugeot and likely Fiat, Mitsubishi, VDO-types.
 
+Fcc-Id:
```

Comment:
Is this meant as a hint for someone to go hunting for it? It doesn't really help if it's empty...